tests/test_ecephys/test_ecephys_interfaces.py

```diff
@@ -49,46 +48,17 @@ def test_ced_import_assertions_3_11(self):
             CEDRecordingInterface.get_all_channels_info(file_path="does_not_matter.smrx")
 
 
-def test_pkl_interface():
-    toy_data = se.example_datasets.toy_example()
-    test_dir = Path(mkdtemp())
-    output_folder = test_dir / "test_pkl"
-    nwbfile_path = str(test_dir / "test_pkl_files.nwb")
-
-    se.save_si_object(object_name="test_recording", si_object=toy_data[0], output_folder=output_folder)
-    se.save_si_object(object_name="test_sorting", si_object=toy_data[1], output_folder=output_folder)
-
-    class SpikeInterfaceTestNWBConverter(NWBConverter):
-        data_interface_classes = dict(Recording=SIPickleRecordingInterface, Sorting=SIPickleSortingInterface)
-
-    source_data = dict(
-        Recording=dict(file_path=str(test_dir / "test_pkl" / "test_recording.pkl")),
-        Sorting=dict(file_path=str(test_dir / "test_pkl" / "test_sorting.pkl")),
-    )
-    converter = SpikeInterfaceTestNWBConverter(source_data=source_data)
-    metadata = converter.get_metadata()
-    metadata["NWBFile"]["session_start_time"] = datetime.now().astimezone()
-    converter.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata, overwrite=True)
-
-    nwb_recording = se.NwbRecordingExtractor(file_path=nwbfile_path)
-    nwb_sorting = se.NwbSortingExtractor(file_path=nwbfile_path)
-    check_recordings_equal(RX1=toy_data[0], RX2=nwb_recording)
-    check_recordings_equal(RX1=toy_data[0], RX2=nwb_recording, return_scaled=False)
-    check_sortings_equal(SX1=toy_data[1], SX2=nwb_sorting)
-
-
 class TestSortingInterface(unittest.TestCase):
     def setUp(self) -> None:
         self.sorting_start_frames = [100, 200, 300]
         self.num_frames = 1000
-        sorting = se.NumpySortingExtractor()
-        sorting.set_sampling_frequency(3000)
+        sorting = NumpySorting(sampling_frequency=3000.0)
         sorting.add_unit(unit_id=1, times=np.arange(self.sorting_start_frames[0], self.num_frames))
         sorting.add_unit(unit_id=2, times=np.arange(self.sorting_start_frames[1], self.num_frames))
         sorting.add_unit(unit_id=3, times=np.arange(self.sorting_start_frames[2], self.num_frames))
```

Comment:
@alejoe91 This is the part I need a little help on - there's no more 'add unit' function for `NumpySorting` - probably easier/faster if you can modify as needed for these couple of tests

Reply:
Sure thing! On it :)

Reply:
Done in last commit! @CodyCBakerPhD let me know if you need help for other parts ;)

Reply:
@alejoe91 Thanks a bunch friend

Wow, the way you add units sure has changed from the old days